lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-simple-permalink.tpl.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Reportback confirmation/permalink page for Rogue.
+ */
+?>
+<article class="reportback__permalink">
+  <header role="banner" class="header ">
+    <div class="wrapper">
+      <h1 class="header__title">Awesome! We got your photo.</h1>
+      <p class="header__subtitle">You've entered into the scholarship and your photo will show up in the gallery soon!</p>
```

Comment:
Actually, we might just want to scrap the scholarship lang here? 

I'm not sure if we can tell if there is a scholarship or not at this point. The only information we have on this page is the Rogue signup id and the Rogue post id so we can't tell what campaign it belongs to (and thus, can't check to see if the campaign has a scholarship or not).